src/static/css/page.css

```diff
@@ -780,3 +780,20 @@ pre {
 .code-block code {
   width: 100%;
 }
+
+/*
+ * Automaticaly adds language after an anchor, depending on its URL
+ * or the content of its `hreflang` attribute (should only be
+ * used when different lang).
+ *
+ * The first three lines is a fallback for those
+ * that haven’t implemented `hreflang`.
+ */
+html:not([lang="en"]) [href*="/en-US/"]::after,
+html:not([lang="en"]) [href*="/en/"]::after,
+html:not([lang="en"]) [href^="https://en."]::after,
+[hreflang]::after {
+  content: '(' attr(hreflang) ')';
```

Comment:
This won’t work for the fallbacks as they won’t have a `hreflang`.  Which is why I think
You need two sets of rules - with the fallback hardcoded to `en`

Reply:
Oh damn, I'll modify this tomorrow, you're right.

Reply:
Done